applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -168,24 +204,41 @@ static int adv_create(void)
 		LOG_ERR("Failed to set periodic advertising parameters (ret %d)", ret);
 		return ret;
 	}
-	ret = bt_audio_broadcast_source_get_id(broadcast_source, &broadcast_id);
-	if (ret) {
-		LOG_ERR("Unable to get broadcast ID: %d", ret);
-		return ret;
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_ID_RANDOM)) {
+		ret = bt_audio_broadcast_source_get_id(broadcast_source, &broadcast_id);
+		if (ret) {
+			LOG_ERR("Unable to get broadcast ID: %d", ret);
+			return ret;
+		}
+	} else {
+		broadcast_id = CONFIG_BT_AUDIO_BROADCAST_ID_FIXED;
 	}
 
 	/* Setup extended advertising data */
 	net_buf_simple_add_le16(&ad_buf, BT_UUID_BROADCAST_AUDIO_VAL);
 	net_buf_simple_add_le24(&ad_buf, broadcast_id);
 
 	ext_ad[0].type = BT_DATA_BROADCAST_NAME;
-	ext_ad[0].data = name;
-	ext_ad[0].data_len = strlen(name);
+	ext_ad[0].data = CONFIG_BT_AUDIO_BROADCAST_NAME;
```

Comment:
Maybe we could use the macro BT_DATA for making the code easier to read.
https://docs.zephyrproject.org/apidoc/latest/group__bt__gap.html#ga8481217e632522e1f322de87d745f8f0
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/peripheral_lbs/src/main.c#L43